static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -190,40 +189,40 @@ function useReplayData({replaySlug, orgSlug}: Options): Result {
   );
 
   const fetchErrors = useCallback(
-    async errorIds => {
-      const conditions = new MutableSearch('');
-      errorIds.forEach((errorId, i) => {
-        if (i > 0) {
-          conditions.addOp('OR');
-        }
-        conditions.addFilterValues('id', [errorId]);
-      });
+    async (replayRecord: ReplayRecord) => {
+      if (!replayRecord.errorIds.length) {
+        return [];
+      }
 
       const response = await api.requestPromise(`/organizations/${orgSlug}/events/`, {
         query: {
           field: ['id', 'error.value', 'timestamp', 'error.type', 'issue.id'],
-          projects: [projectSlug],
-          query: conditions.formatString(),
+          projects: [-1],
+          start: replayRecord.startedAt.toISOString(),
+          end: replayRecord.finishedAt.toISOString(),
+          query: `id:[${String(replayRecord.errorIds)}]`,
+          referrer: 'api.replay.details-page',
```

Comment:
This gets put together to be something like:
![Screen Shot 2022-08-30 at 1 34 38 PM](https://user-images.githubusercontent.com/187460/187538019-d2e14fa3-924b-4f51-ac05-e2f97121dc6a.png)

```
Request URL: https://dev.getsentry.net:7999/api/0/organizations/sentry/events/?end=2022-08-30T19%3A37%3A11.000Z&field=id&field=error.value&field=timestamp&field=error.type&field=issue.id&projects=-1&query=id%3A%5B8976b9b3376e4e4f9c4c278495c32c22%2C9e4d685eb3c04ad092cf97701141c86d%2C375f5bd24261499bae24ccd5e7f8e50c%2Cd7af866f58454802b5aa8bbd4c9f1eba%5D&referrer=api.replay.details-page&start=2022-08-30T18%3A22%3A47.000Z
```